LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+/**
```

Comment:
Remove the autogenerated file? This isn't _our_ code's license, it's listing licenses for other libraries we're using.